src/sentry/mediators/sentry_app_installations/notifier.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from sentry.http import safe_urlopen, safe_urlread
+from sentry.mediators import Mediator, Param
+from sentry.utils.cache import memoize
+
+
+class Notifier(Mediator):
```

Comment:
Just to make sure I understand is notifier _only_ going to be used for the setup installation webhook? vs where we would send events that ppl subscribe to?

Reply:
Yea, the name is too vague. I think we'll want to think about rolling this into the broader Service Hook work we do in the future, but I just encapsulated it here for now.

I'll change the name to something more specific for now.